.github/workflows/wheels.yml

```diff
@@ -29,11 +29,18 @@ jobs:
       with:
         submodules: true
 
+    - name: Install uv
+      uses: astral-sh/setup-uv@v4
+      with:
+        version: '0.5.5'
+        enable-cache: true
+        cache-dependency-glob: "uv.lock"
+
     - name: Build SDist
-      run: pipx run build --sdist
+      run: uv build --sdist
```

Comment:
Weird we're doing this in two steps:
1. Does the `sdist` work? Do we supply binaries of the cpp content? etc...
2. Do we need `sdist` at all